app/src/main/java/com/example/poste/api/poste/models/FolderAccess.java

```diff
@@ -16,9 +16,9 @@ public enum FolderAccess {
 
     public static FolderAccess valueOf(int value) {
         switch (value) {
-            case 1: return VIEW;
-            case 2: return MANAGE;
-            case 3: return OWNER;
+            case 1: return VIEWER;
+            case 2: return EDITOR;
+            case 3: return FULL_ACCESS;
             default: return NONE;
```

Comment:
--Non-blocking Question--
I Don't think this hurts anything, but I thought Folder access no longer used this class. I thought it was changed to just a simple string or something. maybe I'm wrong?
either way no harm in this code.

Reply:
These are just enums, so I don't see the harm here. It might be more clear to use the models or utils classes we've moved functionality to, though.